### PR TITLE
feat: added new actions

### PR DESCRIPTION
**Pull request by cazala**
This PR adds new actions:
- Teleport Player
- Move Player
- Play Default Emote
- Play Custom Emote
- Open Link

The Move Player and Play Custom Emote are not enabled in the inspector because they don't work as expected, they have bugs that need to be fixed in the SDK, but the code in this repo is OK. I left the console.logs for those two actions for debugging purposes until we fix them.

### CODE REVIEW

File: src/definitions.ts [hunk 1]
```diff
@@ -72,6 +72,34 @@ export const ActionSchemas = {
     volume: Schemas.Optional(Schemas.Float),
   }),
   [ActionType.STOP_AUDIO_STREAM]: Schemas.Map({}),
+  [ActionType.TELEPORT_PLAYER]: Schemas.Map({
+    x: Schemas.Int,
+    y: Schemas.Int,
+  }),
+  [ActionType.MOVE_PLAYER]: Schemas.Map({
+    position: Schemas.Map({
```
Comment (cyaiox) [inline src/definitions.ts:80]:
```suggestion
    position: Schemas.Vector3,
```

File: src/actions.ts [hunk 2]
```diff
@@ -452,6 +491,60 @@ export function createActionsSystem(
...
+    void triggerEmote({ predefinedEmote: emote })
+  }
+
+  // PLAY CUSTOM EMOTE
+  function handlePlayCustomEmote(
+    _entity: Entity,
+    payload: ActionPayload<ActionType.PLAY_CUSTOM_EMOTE>,
+  ) {
+    const { src, loop } = payload
+    console.log('triggerSceneEmote', payload)
```
Comment (cyaiox) [inline src/actions.ts:536]: is this `console.log` required for debugging too?

Comment (cazala) [reply src/actions.ts]: i removed that one because we already know the issue is within the triggerSceneEmote

File: src/definitions.ts (hunk 1 above)
Comment (cazala) [reply src/definitions.ts]: done ✅

**Review by cyaiox — APPROVED** [review]

**Pull request merged**